web/src/main/webapp/WEB-INF/config-lucene.xml

```diff
@@ -183,6 +183,7 @@
       <field name="standardName" tagName="standardName"/>
       <field name="_groupWebsite" tagName="groupWebsite"/>
       <field name="attributeTable" tagName="attributeTable"/>
+      <field name="featureTypes" tagName="featureTypes"/>
```

Comment:
You removed `attributeTable` from the indexing and replaced it by this one, so you can remove it from the dumpfield too.

Reply:
done by https://github.com/geonetwork/core-geonetwork/pull/3951/commits/82c57808a09057fa8493c3bbcf4701c99b764a43